v1/schema.graphqls

```diff
@@ -0,0 +1,96 @@
+"""
+A user can have one of many different statuses. This will change how they are displayed.
+"""
+enum Status {
+  HIDDEN
+  AWAY
+  IN_TUTORIUM
+  IN_LESSON
+  PRESENT
+}
+
+"""
+A user's position relative to some fixed point on the map.
+The floor number indicates the level of the building the position is pointing to.
+A floor value of 0 means ground level.
+"""
+type Position {
+  x: Float!
+  y: Float!
+  floor: Int!
```

Comment:
Are you all okay with this or should we do something else (figuring out the floor from GPS coordinates will not be super easy but should be doable). An alternative would be to just use a float for z direction as well.

Reply:
Seems like the best option here. I assume the GPS will be able to determine the integer designated for a floor. A float does not seem necessary.